client/src/assets/islands/locked/Cloud2.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!-- Generator: Gravit.io --><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="isolation:isolate" viewBox="1570.373 578.5 526 289" width="526pt" height="289pt"><g><path d=" M 2094.873 724 L 1938.873 672 L 1860.873 710 L 1902.873 850 L 2044.873 810 L 2094.873 724 Z " fill="rgb(255,255,255)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(255,255,255)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1938.873 828 L 1902.873 850 L 1841.873 850 L 1800.873 866 L 1721.873 827 L 1632.873 810 L 1632.873 754 L 1724.873 754 L 1938.873 828 Z " fill="rgb(195,195,195)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(195,195,195)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1571.873 755 L 1632.873 787 L 1632.873 755 L 1721.873 755 L 1857.873 711 L 1877.873 627 L 1849.581 615.731 L 1759.873 580 L 1701.873 673 L 1632.873 701 L 1571.873 755 Z " fill="rgb(255,255,255)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(255,255,255)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1891.873 606 L 1944.873 620 L 1961.873 673 L 2061.873 711 L 1944.873 701 L 1919.873 682 L 1857.873 711 L 1877.873 627 L 1891.873 606 Z " fill="rgb(232,232,232)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(232,232,232)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1730.873 627 L 1704.873 754 L 1654.873 692 L 1701.873 673 L 1730.873 627 Z " fill="rgb(232,232,232)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(232,232,232)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1724.873 753 L 1902.873 827 L 1860.873 709 L 1724.873 753 Z " fill="rgb(247,245,245)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(247,245,245)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/><path d=" M 1841.873 606 L 1895.873 594 L 1969.873 606 L 2006.873 655 L 2038.873 701 L 1961.873 673 L 1944.873 620 L 1891.873 606 L 1877.873 627 L 1824.873 606 L 1841.873 606 Z " fill="rgb(255,255,255)" vector-effect="non-scaling-stroke" stroke-width="1" stroke="rgb(255,255,255)" stroke-linejoin="miter" stroke-linecap="square" stroke-miterlimit="3"/></g></svg>
```

Comment:
I think we shouldn't capitalize it as the other files. `cloud2.svg` instead of `Cloud2.svg`